apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -366,16 +369,38 @@ export default class CoreLibrary {
     return speechBubbles[speechBubbles.length - 1];
   }
 
+  // When the total number of sprites reaches MAX_NUM_SPRITE_WARNING,
+  // a workspace alert warning is displayed.
+  // MAX_NUM_SPRITE_WARNING = MAX_NUM_SPRITES - 1
+  reachedSpriteLimit() {
+    const numSprites = this.getNumberOfSprites();
+    if (numSprites > MAX_NUM_SPRITE_WARNING) {
+      return true;
+    }
+    if (numSprites === MAX_NUM_SPRITE_WARNING) {
```

Comment:
For my own understanding: Do we have the `=== MAX_NUM_SPRITE_WARNING` so the warning gets displayed only once per time the user goes over the max number of sprites? i.e. if the user puts 1005, the warning is displayed once instead of 5x?

Reply:
Yes, that's correct.

Reply:
(i.e. as opposed to using `>= MAX_NUM_SPRITES`)

Reply:
(oops I had the page cached and didn't see your response. Ignore my second comment)